lib/blob/cluster/cluster_test.go

```diff
@@ -164,12 +164,11 @@ func (s *ClusterMultiPeers) SetUpTest(c *C) {
 			Clock:            fakeClock,
 			ID:               fmt.Sprintf("%v", i),
 			AdvertiseAddr:    "https://localhost",
-			TestMode:         true,
 			GracePeriod:      gracePeriod,
 		})
 		c.Assert(err, IsNil)
-		objects[i] = obj.(*cluster)
-		c.Assert(objects[i].heartbeat(), IsNil)
+		objects[i] = obj
+		objects[i].heartbeat()
```

Comment:
Is there a reason for not asserting heartbeat returns nil anymore?

Reply:
That's a good question - not that I see any. I'll revert this line.